### PR TITLE
feature(templates) Add extra template for feature doc requests

### PR DESCRIPTION
**Pull request by EugeneHlushko**
As described here, we want to make it mandatory during the feature introduction to at least file an issue in our docs webpack/webpack#7267

### CODE REVIEW

**Comment by ooflorent** [comment]
I could be great to add metadata to the template to display the selection screen like in webpack:

issues

**Comment by EugeneHlushko** [comment]
hmm thought this was automatic by GH 😭 i will add, thanks 👍

File: .github/ISSUE_TEMPLATES/documentation_request.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+---
+name: Documentation request
+about: Create a documentation request for a new feature or functioanlity change.
```
Comment (montogeek) [inline .github/ISSUE_TEMPLATES/documentation_request.md]: functionality typo

Comment (EugeneHlushko) [reply .github/ISSUE_TEMPLATES/documentation_request.md]: thanks man, missed it

**Comment by EugeneHlushko** [comment]
@montogeek what do you think about this as the bot solution was preferred for webpack/webpack repo. maybe we still want this in but extended into few possible issue templates?

The one i've added LGTM plus we had the original. Maybe we can think of another frequently used issue submission to improve this flow?

**Comment by montogeek** [comment]
I like bot solution, I still don't understand how it knows a webpack change merits a doc update as well

**Comment by montogeek** [comment]
It is done manually

**Comment by EugeneHlushko** [comment]
yeah its done manualy via checkbox, basically bot puts a message for reviewer that contains a checkbox that if ticked by reviewer will result in an issue here after original PR is merged

**Comment by EugeneHlushko** [comment]
not sure if we keep this PR or not, any thoughts?

**Comment by montogeek** [comment]
Yes, people still can post issues here.

**Comment by EugeneHlushko** [comment]
ok cool lets get more reviews then CC @jeremenichelli

**Comment by EugeneHlushko** [comment]
@jeremenichelli i did remove the white space, nice catch i just hope it wasnt there for a reason as i copied meta block from webpack repo. The .md already ends with new line dunno why git didnt show it, linter would fail ^^

**Comment by EugeneHlushko** [comment]
@jeremenichelli @montogeek is this still a thing or should i close it?

**Comment by montogeek** [comment]
Still

**Comment by jeremenichelli** [comment]
@EugeneHlushko I think this is a good addition if we can add the metadata as @ooflorent mentioned, if not most people will avoid using the templates I think. But would love to have it honestly.

**Comment by jeremenichelli** [comment]
@EugeneHlushko I think this is a good addition if we can add the metadata as @ooflorent mentioned, if not most people will avoid using the templates I think. But would love to have it honestly.

**Comment by EugeneHlushko** [comment]
@jeremenichelli but metadata is there already, the thing is there we will have only one template and a default empty issue, while webpack has 3-4 options

**Comment by jeremenichelli** [comment]
Then let's give it a go :)

**Comment by EugeneHlushko** [comment]
ok, updating this one. CC @montogeek

**Comment by montogeek** [comment]
Thanks!

**Pull request merged**